pandas/core/groupby/groupby.py

```diff
@@ -3228,39 +3231,31 @@ def ngroup(self, ascending: bool = True):
 
         Examples
         --------
-        >>> df = pd.DataFrame({"A": list("aaabba")})
+        >>> df = pd.DataFrame({"color": ["red", None, "red", "blue", "blue", "red"]})
         >>> df
-           A
-        0  a
-        1  a
-        2  a
-        3  b
-        4  b
-        5  a
-        >>> df.groupby('A').ngroup()
+           color
+        0    red
+        1   None
+        2    red
+        3   blue
+        4   blue
+        5    red
+        >>> df.groupby("color").ngroup()
+        0    0.0
+        1    NaN
+        2    0.0
+        3    1.0
+        4    1.0
+        5    0.0
+        dtype: float64
+        >>> df.groupby("color", dropna=False).ngroup()
```

Comment:
I dropped the `ascending` example, I figured that was obvious enough. Think I should keep it in?

Reply:
I think we typically demonstrate all the features of a method, regardless of how obvious they might seem. I think it should be retained.

Reply:
I restored
```
        >>> df.groupby("color", dropna=False).ngroup(ascending=False)
        0    1
        1    0
        2    1
        3    2
        4    2
        5    1
        dtype: int64
```

I chose to use `dropna=False` because I wanted to show
that NA keys are placed BEFORE other keys.

I figured the `dropna=True` example was obvious enough from this and
I didn't need that one as well.